devices/SPS30/Entities/Measurement.cs

```diff
@@ -0,0 +1,72 @@
+﻿//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace Iot.Device.SPS30.Entities
+{
+    /// <summary>
+    /// Abstract Measurement class that can house both response types (Float vs UInt16) by using doubles.
+    /// </summary>
+    public abstract class Measurement
```

Comment:
Question regarding the design here:
* You may not need this abstract and just use a constructor passing the MeasurementOutputFormat and then processing the conversion according to it.
* it would remove the 2 MeasearumentFloat and MeasurementUInt16
* Just add the enum as a property like for the rest

This would simplify the implementation and the readability without changing at all the functionality.

Thoughts?

Reply:
Agreed. It'll keep the parsing more contained.